src/main/java/at/hannibal2/skyhanni/features/event/hoppity/HoppityAPI.kt

```diff
@@ -183,15 +183,18 @@ object HoppityAPI {
         return (month % 2 == 1) == (day % 2 == 0)
     }
 
-    private fun Map<Int, ItemStack>.filterStrayProcessable() = filter { (slotNumber, stack) ->
+    fun Map<Int, ItemStack>.filterMayBeStray() = filter { (slotIndex, stack) ->
```

Comment:
Only generic utils funcitons should  be public and extended. this is a limited scope utils function, and we dont want to offer this funciton as option to tab complete in other features of the mod.
simplest solution: dont make a extended funciton here